.travis.yml

```diff
@@ -13,56 +13,83 @@ notifications:
 cache:
   directories:
     - $HOME/.composer/cache
-    - $HOME/.phpbrew
+    - $HOME/.jest-cache
     - $HOME/.npm
-
-before_install:
-  - nvm install && nvm use
-  - npm install npm -g
+    - $HOME/.nvm/.cache
+    - $HOME/.phpbrew
 
 branches:
   only:
     - master
 
+before_install:
+  - nvm install
+
 jobs:
   include:
-    - stage: test
+    - name: JS unit tests
       env: WP_VERSION=latest
+      before_install:
+        - nvm install --latest-npm
```

Comment:
Out of curiosity, why is it needed here? Does it override `nvm install` from line 26?

Reply:
Yes defining `before_install` again overrides the previous `before_install`. We need `--latest-npm` here so that we're using the latest version of `npm`. We don't need `--latest-npm` elsewhere because `./bin/setup-local-env.sh` will upgrade `npm` if necessary.